spec/unit/blockscore/candidate_spec.rb

```diff
@@ -1,46 +1,192 @@
+
+      it { is_expected.to be_persisted }
+      it { expect(candidate.name_first).not_to be_empty }
+      it { expect(candidate.class).to be BlockScore::Candidate }
+    end
 
-    let(:candidate) { create(:candidate_params) }
+    # NB: At the time of writing, the testing API does not reset. Making this
+    #     test less then perfect. To work around the near certain availability
+    #     of historical candidates from other tests known candidates are created
+    #     and checked. This has the side effect of being susceptible to race
+    #     conditions if tested against the live API in parallel.
+    describe '.all' do
+      let(:uniq_token_one) { '05816347d2234ef4a92465e9784c7ce1' }
+      let(:uniq_token_two) { 'b4cd5fb645788c3b37c11407b83d7741' }
+      subject(:candidates) { BlockScore::Candidate.all }
 
-    context 'updating' do
       before do
-        candidate.name_first = 'Chris'
-        candidate.save
+        create(:candidate, name_first: 'John', name_last: uniq_token_one)
+        create(:candidate, name_first: 'John', name_last: uniq_token_two)
+      end
+
+      it { is_expected.not_to be_empty }
+      it { expect(candidates[0].name_last).to eq uniq_token_two }
+      it { expect(candidates[1].name_last).to eq uniq_token_one }
+    end
+
+    describe '#save' do
+      context 'when creating a new candidate' do
+        subject(:candidate) { build(:candidate) }
+        before { candidate.save }
+
+        it { is_expected.to be_persisted }
+        it { expect(candidate.name_first).not_to be_empty }
+      end
+
+      context 'when updating an existing candidate' do
+        subject(:candidate) { create(:candidate, name_first: 'John') }
+        before do
+          candidate.name_first = 'Jane'
+          candidate.save
+        end
+
+        it { is_expected.to be_persisted }
+        it { expect(candidate.name_first).to eq 'Jane' }
+      end
+
+      pending 'after deleting a candidate' do
+        subject(:candidate) { create(:candidate) }
+        before do
+          candidate.delete
+        end
+
+        it { expect { candidate.save }.to raise_error }
       end
+    end
 
-      let(:uri)         { "/candidates/#{candidate.id}"                     }
-      let(:http_method) { :patch                                            }
-      let(:expected)    { { body: hash_including('name_first' => 'Chris') } }
+    describe '#refresh' do
+      subject(:candidate) { create(:candidate, name_last: 'Smith') }
+      before do
+        candidate.name_last = 'John'
+        candidate.refresh
+      end
 
-      it { expect(candidate.name_first).to eql('Chris') }
-      include_context 'api request'
+      it { expect(candidate.name_last).to eq 'Smith' }
     end
 
-    describe '#delete' do
-      before { candidate.delete }
+    describe '#inspect' do
+      subject(:candidate_inspection) { create(:candidate).inspect }
+      it { expect(candidate_inspection.class).to be String }
+    end
```

Comment:
I usually tell my coverage tools to ignore `#inspect` methods I customize.  I'm fine with specifying a test, but in this case we might as well assert it matches some format instead of just asserting it is a string